ql/src/test/results/clientpositive/llap/vector_decimal_aggregate.q.out

```diff
@@ -847,4 +847,4 @@ FROM (SELECT cint,
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@decimal_vgby_small
 #### A masked pattern was here ####
-95165244160
+95767761728
```

Comment:
here also the result of the query is changing

Reply:
This difference is caused by the previous differences in STDDEV_POP(cdecimal1) and STDDEV_SAMP(cdecimal1) fields